force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,36 +212,37 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+       
+        String result = '';   // Initialize the result String
+
+        for (Integer i = 1; i <= repeatCount; i++) {    // Use a for loop to append the inputString to the result string repeatCount times
+            result += inputString;      
+            if (i < repeatCount) {   // If it is not the last iteration, add a semicolon to separate the strings
+            result += '; ';
+            }
         }
-        */
 
-        // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return result;
+
     }
 
+
     /**
      * Question 11
      * Creates a List of integers and adds the integers 1, 2, and 3 to it.
      * Example: createAndPopulateList() should return [1, 2, 3]
      * @return A List of integers containing the numbers 1, 2, and 3.
      */
     public static List<Integer> createAndPopulateList() {
-        // Create a new list of integers
-
-        // Add the integers 1, 2, and 3 to the list
-        // add 1
-        // add 2
-        // add 3
+        
+        List<Integer> myList = new List<Integer>();
+        
+        myList.add(1);
+        myList.add(2);
+        myList.add(3);
+        
+        return myList; 
```

Comment:
Great work. You can also use the List constructor and save yourself a few lines of code

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
    ```